### ISSUE TITLE
Context ID is unknown until event is triggered on StreamDeck

### ISSUE THREAD

**Issue by avarabyeu** [ybizeul/StreamDeckWS#39]
Hey. First of all, thx for an amazing plugin. 
I was wondering if there's any chance to receive context ID (aka uuid) when websocket connection is established. 
My use case: stream deck is connected and websocket server is started afterwards (not a node red, custom one). After ws connection is established, there's no way to send an event pro-actively since context ID is unknown. 
It would be nice to receive it as http header or have a possibility to request it somehow for given remote server

**Comment by ybizeul** [comment]
Hi, if I got your question correctly, I think the answer is that you can't do this as you describe it, the API is called only when something is happening, and that's what I'm forwarding alongside the context ID.

That said, when streamdeck starts, it pushes an event for every key, that's your chance to get thir context id and cache it. It's the method I use in the node red plugin to learn all the keys so you can push an arbitrary event whenever you want. 

In other words, run your WS server, and restart streamdeck software, you should see willAppear event for each key.
